topics/ui/index.md

```diff
@@ -0,0 +1,10 @@
+---
+aliases: user-interface
+created_by: Glenda Schroeder
```

Comment:
This field is meant to be used for the creator of the subject of the topic, in this case `ui`. I do not see `Glenda Schroeder` anywhere on the wikipedia page _and_ I think this topic is too broad to have any `created_by` attribute. Can we remove this altogether?
```suggestion
```